python/ray/experimental/workflow/api.py

```diff
@@ -8,16 +9,52 @@
 # avoid collision with arguments & APIs
 from ray.experimental.workflow import virtual_actor_class
 from ray.experimental.workflow import storage as storage_base
+from ray.experimental.workflow.storage import Storage
+from ray.experimental.workflow import workflow_access
 
 if TYPE_CHECKING:
-    from ray.experimental.workflow.storage import Storage
     from ray.experimental.workflow.virtual_actor_class import (
         VirtualActorClass, VirtualActor)
 
 logger = logging.getLogger(__name__)
 
-# TODO(suquark): Update the storage interface (e.g., passing it through
-# context instead of argument).
+
+def init(storage: "Optional[Union[str, Storage]]" = None) -> None:
+    """Initialize workflow.
+
+    Args:
+        storage: The external storage URL or a custom storage class. If not
+            specified, ``/tmp/ray/workflow_data`` will be used.
+    """
+    if storage is None:
+        storage = os.environ.get("RAY_WORKFLOW_STORAGE")
+    if storage is None:
+        # We should use get_temp_dir_path, but for ray client, we don't
+        # have this one. We need a flag to tell whether it's a client
+        # or a driver to use the right dir.
+        # For now, just use /tmp/ray/workflow_data
+        logger.warning("Using default local dir: `/tmp/ray/workflow_data`. "
+                       "This should only be used for testing purposes.")
+        storage = "file:///tmp/ray/workflow_data"
+    if isinstance(storage, str):
+        storage = storage_base.create_storage(storage)
+    elif not isinstance(storage, Storage):
+        raise TypeError("'storage' should be None, str, or Storage type.")
+    try:
+        _storage = storage_base.get_global_storage()
+    except RuntimeError:
+        pass
+    else:
```

Comment:
Add some "empty" lines to separate the blocks for easy understanding. It's a little bit hard to distinguish the blocks here.